lib/engine.js

```diff
@@ -102,7 +106,13 @@ class Engine {
      * @returns 
      */
     wrapError(sourceError, message, location) {
-        if (!(sourceError instanceof GenericError) && !(sourceError instanceof ClientError)) {
+        if (!(sourceError instanceof GenericError) 
+            && !(sourceError instanceof ClientError)
```

Comment:
another idea, we can compare strings `error.reason === Reason.RenditionFormatUnsupportedError`: https://github.com/adobe/asset-compute-commons/blob/master/lib/errors.js#L18-L25